lib/hubspot/blog.rb

```diff
@@ -73,6 +73,7 @@ def self.find_by_blog_post_id(id)
 
     def initialize(response_hash)
       @properties = response_hash #no need to parse anything, we have properties
+      @properties['topic_names'] = topics.collect { |t| t['name'] }
```

Comment:
Any reason to not fold this into the `topics` method that already below?

Reply:
Yes. That's what I'd prefer to do. (This is what I was trying to explain earlier, but wasn't doing a good job.) 

In the Jekyll gem, when you grab the appropriate property, you are [using the `properties` instance variable](https://github.com/crdschurch/jekyll-hubspot/blob/master/lib/jekyll-hubspot/document.rb#L90).

Because this is a variable and not a method, I'd have to find a way to inject the property into `@properties`.

I'd rather use a `topic_names` method, but for that to work with the Jekyll gem, we'd then have to check instance methods in addition to property keys. I don't mind that approach (I actually like it because it feels cleaner and more ruby-esque). But it's a bit more complicated and felt like over-engineering.

I'd love to be cleaner about this, though, so if you have an idea on how we could get the instance method approach to work, I'm all ears.